student/src/main/java/tds/student/sql/repository/RemoteExamRepository.java

```diff
@@ -98,6 +99,15 @@ private NoContentResponseResource handleErrorResponseNoContent(String body) thro
     }
   }
   
+  private Response<ExamConfiguration> handleErrorResponseExamConfig(String body) throws ReturnStatusException {
+    try {
+      JavaType type = objectMapper.getTypeFactory().constructParametricType(Response.class, Exam.class);
+      return objectMapper.readValue(body, type);
+    } catch (IOException e) {
+      throw new ReturnStatusException(e);
```

Comment:
That's probably a good idea.  I didn't do that in the other place we have similar code and should've done that.